apps/exports/models.py

```diff
@@ -45,6 +45,11 @@ def last_run(self):
     def is_scheduled_to_run(self):
         return export_is_scheduled_to_run(self, self.last_run)
 
+    def has_queued_runs(self):
+        if self.runs.exists():
+            # queued runs that aren't the latest should be ignored so just check the latest one
```

Comment:
My thinking is that if the latest run is already queued up there's no need to ever queue up another one (since each run ingests whatever new data is available having multiple run sequentially doesn't really add any value)